src/ModelerNode.js

```diff
@@ -0,0 +1,60 @@
+import { g } from 'jointjs';
+
+export default class ModelerNode {
+  constructor(type, nodeRegistry, moddle, $t) {
+    this.type = type;
+    this.definition = nodeRegistry[type].definition(moddle, $t);
+    this.diagram = nodeRegistry[type].diagram(moddle);
+  }
+
+  isType(bpmnType) {
+    return this.bpmnType === bpmnType;
+  }
+
+  get bpmnType() {
+    return this.definition.$type;
+  }
+
+  getSize() {
+    return {
+      width: this.diagram.bounds.width,
+      height: this.diagram.bounds.height,
+    };
+  }
+
+  setPosition({ x, y }) {
```

Comment:
I think both set and get position should use Javascript getter/setter functions.